autonomy/cli/deploy.py

```diff
@@ -272,6 +280,26 @@ def run_deployment_from_token(  # pylint: disable=too-many-arguments, too-many-l
     run_deployment(build_dir)
 
 
+def update_multisig_address(service_path: Path, address: str) -> None:
+    """Update the multisig address on the service config."""
+
+    with open_file(service_path / DEFAULT_SERVICE_FILE) as fp:
+        config, *overrides = yaml_load_all(
+            fp,
+        )
+
+    for override in overrides:
+        if override["type"] == SKILL:
+            addresses = override["setup_args"]["args"]["setup"]["safe_contract_address"]
+            override["setup_args"]["args"]["setup"]["safe_contract_address"] = [
+                *addresses,
+                address,
+            ]
+
+    with open_file(service_path / DEFAULT_SERVICE_FILE, mode="w+") as fp:
+        yaml_dump_all([config, *overrides], fp)
```

Comment:
This was not supposed to be part of this PR, I'll revert this and create a proper PR later. This is just here for the context